avogadro/mainwindow.cpp

```diff
@@ -1627,6 +1628,18 @@ void MainWindow::setBackgroundColor()
   }
 }
 
+void MainWindow::setRenderingSettings()
+{
+  Rendering::SolidPipeline *pipeline(nullptr);
+  GLWidget *glWidget(nullptr);
+  if ((glWidget = qobject_cast<GLWidget *>(m_multiViewWidget->activeWidget())))
+    pipeline = &glWidget->renderer().solidPipeline();
+  if (pipeline) {
+    RenderingDialog dialog(this, *pipeline);
+    dialog.exec();
+  }
```

Comment:
Right here, you should use `QSettings::setValue()` to save the user choices for parameters.